tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -94,6 +94,22 @@ static uint8_t dev_uuid[16] = { 0x6c, 0x69, 0x6e, 0x67, 0x61, 0x6f };
 static uint8_t *uuid_to_provision;
 static struct k_sem reprov_sem;
 static uint32_t link_close_timestamp;
+static bt_mesh_prov_bearer_t prov_bearer = BT_MESH_PROV_ADV;
```

Comment:
I think this leads to a problem where we may run tests for PB-ADV while thinking that we are running them for PB-GATT. Maybe revert the suffix to the test cases (`pb_adv`, `pb_gatt`) instead of using arguments? Or set `prov_bearer` to 0 and pass `-argstest prov-brearer=1` to pb-adv tests. So we at least know that the argument is parsed. But different names for test cases seems to be better option.